### PR TITLE
test: add tests for content sub-block in Banner

### PR DESCRIPTION
**Pull request by niktverd**

### CODE REVIEW

File: test-utils/shared/content.tsx [hunk 1]
```diff
@@ -0,0 +1,150 @@
...
+};
+
+export const testContentWithList = <T,>({
+    component: Component,
+    props,
+    options,
+}: ContentTestFunction<T>) => {
+    if (
+        !options?.qaId ||
+        !props.list?.[0]?.icon ||
```
Comment (NikitaCG) [inline test-utils/shared/content.tsx:134]: `const listElem = props.list?.[0]` ?

File: test-utils/shared/content.tsx [hunk 2]
```diff
@@ -0,0 +1,150 @@
...
+        throw new Error(ERROR_INPUT_DATA_MESSAGE);
+    }
+
+    const listQa = getQaAttrubutes(options.qaId, ['title', 'text']);
+
+    render(<Component {...pick(props, 'list', 'qa')} />);
+    const image = screen.getByRole('img');
+    const title = screen.getByTestId(listQa.title);
+    const text = screen.getByTestId(listQa.text);
+    expect(image).toHaveAttribute('src', props.list?.[0]?.icon);
```
Comment (NikitaCG) [inline test-utils/shared/content.tsx:147]: without `?`, we did the check above

File: package.json [hunk 3]
```diff
@@ -76,7 +76,7 @@
     "@commitlint/config-conventional": "^17.4.3",
     "@doc-tools/transform": "^3.3.2",
     "@gravity-ui/eslint-config": "^2.0.0",
-    "@gravity-ui/page-constructor": "^4.0.0",
+    "@gravity-ui/page-constructor": "^4.7.0",
```
Comment (NikitaCG) [inline package.json:79]: is  another components are ok after this update?

**Review by NikitaCG — CHANGES_REQUESTED** [review]

File: test-utils/shared/content.tsx (hunk 2 above)
Comment (niktverd) [reply test-utils/shared/content.tsx:147]: Sure

**Review by NikitaCG — APPROVED** [review]

File: package.json (hunk 3 above)
Comment (niktverd) [reply package.json:79]: it looks so

**Pull request merged**